lib/http.js

```diff
@@ -244,13 +244,13 @@ module.exports = function (chai, _) {
    */
 
   Assertion.addProperty('redirect', function() {
-    var redirectCodes = [301, 302, 303]
+    var redirectCodes = [301, 302, 303, 307, 308]
       , status = this._obj.status
       , redirects = this._obj.redirects;
 
     this.assert(
         redirectCodes.indexOf(status) >= 0 || redirects && redirects.length
-      , "expected redirect with 30{1-3} status code but got " + status
+      , "expected redirect status code but got " + status
```

Comment:
yeah that's clearer honestly couldn't think of a good error message so that's a good one